### PR TITLE
Hotfix: Added correct type comparison

### LINKED ISSUE #415: /user Registered Modules bug
Hi!
`/user`in `Registered Module` I see all my repositories, I don't see only registered into site.
It's only a my problem?

Thanks!

### PR DESCRIPTION
**Pull request by ins0**
fixes #415

### CODE REVIEW

**Comment by localheinz** [comment]
Thank you, @ins0!

File: module/ZfModule/src/ZfModule/Service/Module.php [hunk 1]
```diff
@@ -131,7 +131,7 @@ public function currentUserModules()
                 return false;
             }
 
-            if (null === $this->moduleMapper->findByUrl($repository->html_url)) {
+            if (false === $this->moduleMapper->findByUrl($repository->html_url)) {
```
Comment (Ocramius) [inline module/ZfModule/src/ZfModule/Service/Module.php]: Urgh, why is a mapper returning `false` on a non-match? :-(

Comment (localheinz) [inline module/ZfModule/src/ZfModule/Service/Module.php]: @Ocramius 

Wondered myself, but apparently that's what it does - I'd personally expect `null`, or a `NullModule`?!

Comment (localheinz) [inline module/ZfModule/src/ZfModule/Service/Module.php]: @Ocramius 

Here's why: https://github.com/zendframework/zf2/blob/8b644e8b0e8cd931bf25cf3babc3ffa047df26d5/library/Zend/Db/ResultSet/HydratingResultSet.php#L93

Comment (Ocramius) [inline module/ZfModule/src/ZfModule/Service/Module.php]: WTF.

Comment (ins0) [inline module/ZfModule/src/ZfModule/Service/Module.php]: :train: Doctrine :train:

**Pull request merged**